app/controllers/images_controller.rb

```diff
@@ -13,6 +15,13 @@ def create
     redirect_to search_show_path(api_image)
   end
 
+  def download
+    # uuid = SecureRandom.uuid
```

Comment:
Can we remove the comments?

Reply:
I just removed the comments and committed.